ntbs-service/Pages/SearchOld/Index.cshtml

```diff
@@ -13,9 +13,9 @@
     <form method="post">

```

Comment:
Let's move these to a file that describes function better - so migration, not searchOld.

Also, we need to think about either:
- putting in a ticket to remove these.
- or putting this page behind further security
(this might be as part of the follow up ticket if you think that's better placed there)

Reply:
I will create a ticket to remove it then